CHANGELOG.md

```diff
@@ -8,9 +8,9 @@
 
 <!-- next-header -->
```

Comment:
Thanks for the opportunity to see how the release-cutting instructions work in the wild! :-)

Please add a fresh copy of the `UnreleasedVersion` etc header to the top of the file, so there's an obvious place for future changelog entries to be added. Like this, with the new version put into the "changes" link:

```
## [[UnreleasedVersion]] (_[[ReleaseDate]]_)

[All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.11.0...HEAD).
```

In the future I hope that `cargo release` can be configured to do that automatically for us.

Reply:
I would say thank you for such an easy-to-follow document! Everything went pretty smooth except for just adding back the template for next time (and in the future hopefully not even an issue)